editor/components/page-attributes/parent.js

```diff
@@ -81,8 +74,7 @@ const applyWithAPIDataItems = withAPIData( ( props, { type } ) => {
 } );
 
 export default compose( [
-	applyConnect,
-	applyWithAPIDataPostType,
+	applyWithSelect,
+	applyWithDispatch,
 	applyWithAPIDataItems,
-	withInstanceId,
 ] )( PageAttributesParent );
```

Comment:
Out of curiosity, why `withInstanceId` was removed? What is the current status of this HOC?

Reply:
Because it was not used in the component, I think it was a left over.